documentation/src/main/asciidoc/topics/con_write_behind.adoc

```diff
@@ -26,6 +26,15 @@ write to the cache and the write to the cache store is as small as possible.
 include::config_examples/persistence_write_through_async.xml[]
 ----
 
+The preceding configuration example uses the `fail-silently` parameter to
+control what happens when either the cache store is unavailable or the
+modification queue is full.
+
+* If `fail-silently="true"` then {brandname} logs WARN messages and rejects
+write operations.
+* If `fail-silently="false"` then {brandname} throws exceptions and rejects
+write operations.
```

Comment:
This second sentence needs more information. This is what I was clarifying for Wolf. If fail-silently is false, the exceptions are only thrown once the modification queue size becomes full or if  the underlying store is known to be unavailable when a write operation is attempted. For example if the queue size is 100, during the time it takes for the store to be detected as unavailable, it's possible for 99 write operations to be queued without an exception being thrown. If the server is then kill before the store becomes available again, those 99 operations will be lost as they were never persisted.